cni-plugin/internal/pkg/testutils/utils_linux.go

```diff
@@ -43,6 +43,8 @@ import (
 	"github.com/google/uuid"
 	log "github.com/sirupsen/logrus"
 	"github.com/vishvananda/netlink"
+
+	"github.com/projectcalico/calico/libcalico-go/lib/netlinkutils"
```

Comment:
we should keep this import with the rest of the libcalico-go imports (I would expect static-checks to fail and `make fix` to address this?)